lockbox-ios/Action/TelemetryAction.swift

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Telemetry
+import RxSwift
+import RxCocoa
+
+protocol TelemetryAction: Action {
+    var eventMethod: TelemetryEventMethod { get }
+    var eventObject: TelemetryEventObject { get }
+    var value: String? { get }
+    var extras: [String: Any?]? { get }
+}
+
+enum TelemetryEventCategory: String {
+    case action
+}
+
+enum TelemetryEventMethod: String {
+    case tap, signin, startup, foreground, background
+}
+
+enum TelemetryEventObject: String {
+    case app = "app"
+    case entryList = "entry_list"
+    case settingsButton = "settings_button"
```

Comment:
Is there another Issue created for adding in the rest of these events?

Reply:
I think that'll be #58, but you have to fight @irrationalagent for them ;)